substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/Target_sun_security_ssl_TrustStoreManager.java

```diff
@@ -94,27 +109,95 @@ public void afterRegistration(AfterRegistrationAccess access) {
 @TargetClass(className = TrustStoreManagerFeature.TRUST_STORE_MANAGER_CLASS_NAME)
 final class Target_sun_security_ssl_TrustStoreManager {
 
+    @Alias @RecomputeFieldValue(kind = RecomputeFieldValue.Kind.NewInstance, declClassName = TrustStoreManagerFeature.TRUST_STORE_MANAGER_CLASS_NAME +
+                    "$TrustAnchorManager") private static Target_sun_security_ssl_TrustStoreManager_TrustAnchorManager tam = null;
+
     @Substitute
     private static Set<X509Certificate> getTrustedCerts() throws Exception {
-        return ImageSingletons.lookup(TrustStoreManagerSupport.class).trustedCerts;
-    }
+        Set<X509Certificate> set = new HashSet<>(ImageSingletons.lookup(TrustStoreManagerSupport.class).trustedCerts);
+        set.addAll(tam.getTrustedCerts(Target_sun_security_ssl_TrustStoreManager_TrustStoreDescriptor.createInstance()));
 
-    @Substitute
-    private static KeyStore getTrustedKeyStore() throws Exception {
-        return ImageSingletons.lookup(TrustStoreManagerSupport.class).trustedKeyStore;
+        return Collections.unmodifiableSet(set);
     }
 }
 
-/*
- * The internal classes to describe and load root certificates must not be reachable at run time.
- */
-
-@Delete
```

Comment:
This class was deleted for a good reason. It has, e.g., the `java.home` directory of the image builder baked into the static fields `defaultStorePath` and `jsseDefaultStore`. I don't see anything in this PR that would modify usages of these fields

Reply:
But the `TrustAnchorManager` needs a `TrustStoreDescriptor` in order to load the certificate... so maybe I should just invalidate the static fields?